code/app/src/main/java/com/example/sigma_blue/entity/image/ImageManager.java

```diff
@@ -60,21 +61,30 @@ public void updateFromItem(Item item) {
         }
     }
 
+    public ImageListAdapterFromPath getAdapter() {
+        return adapter;
+    }
+
     private void updateFromList() {
         this.entityList.clear();
-        for (String s : pathList) {
-            dbHandler.getImage(s, this::onImageDownload);
+//        for (String s : pathList) {
```

Comment:
Please delete commented out code. If it doesn't get deleted now, we'd have to go back and delete it later.